### DISCUSSION TITLE
Use Appletalk, but problems

### DISCUSSION THREAD

**Discussion by marcuskuenzel1971** [akuker/RASCSI#1105] [General]
Hello,

I successfully installed it and the services are running. But i see no options in the web interface an within mac os.

The status shows:

atalkd.service - Netatalk AppleTalk daemon
     Loaded: loaded (/lib/systemd/system/atalkd.service; enabled; vendor preset: enabled)
     Active: active (running) since Fri 2023-02-24 16:09:50 GMT; 34min ago
      Tasks: 1 (limit: 407)
        CPU: 53ms
     CGroup: /system.slice/atalkd.service
             └─905 /usr/local/sbin/atalkd

Feb 24 16:09:18 piscsi systemd[1]: Starting Netatalk AppleTalk daemon...
Feb 24 16:09:18 piscsi atalkd[905]: Set syslog logging to level: LOG_DEBUG
Feb 24 16:09:18 piscsi atalkd[905]: restart (2.230201)
Feb 24 16:09:19 piscsi atalkd[905]: zip_getnetinfo for wlan0
Feb 24 16:09:29 piscsi atalkd[905]: zip_getnetinfo for wlan0
Feb 24 16:09:39 piscsi atalkd[905]: zip_getnetinfo for wlan0
Feb 24 16:09:49 piscsi atalkd[905]: config for no router
Feb 24 16:09:50 piscsi atalkd[905]: ready 0/0/0
Feb 24 16:09:50 piscsi systemd[1]: Started Netatalk AppleTalk daemon.

So it seems to run, isn't it ?

but i can see it neither in the piscsi web interface nor in mac os.

Thanks,

**Comment by rdmark** [comment]
@marcuskuenzel1971 Can you please describe your network topology here? Is your RPi connected to your LAN over wifi or ethernet? How is your vintage Mac connected to the LAN?

Depending on the topology, you may have to modify the atalkd.conf file to route AppleTalk traffic over one or the other network interface on the RPi Linux system.
